docs/README.md

```diff
@@ -12,7 +12,6 @@ This documentation explains how to use [ReVanced Manager](https://github.com/rev
    3. [🔄 Updating ReVanced Manager](2_3_updating.md)
```

Comment:
Updating has been implemented. The file is out of date (It is also not showing up in the github review page for some reason!)